app/api/helpers/permission_manager.py

```diff
@@ -427,7 +426,10 @@ def permission_manager(view, view_args, view_kwargs, *args, **kwargs):
                 fetched = getattr(data, fetch) if hasattr(data, fetch) else None
 
         if fetched:
-            kwargs[kwargs['fetch_as']] = fetched
+            if 'fetch_as' in kwargs:
+                kwargs[kwargs['fetch_as']] = fetched
+            elif 'fetch' in kwargs:
```

Comment:
Can you explain this change?

Reply:
I am trying to use the value of kwarg ```fetch``` as the default for ```fetch_as```. So if we don't provide the value of ```fetch_as```, we will assume it's the same as ```fetch```